nuitka/build/Backend.scons

```diff
@@ -736,33 +743,22 @@ elif not module_mode:
     python_lib_path = os.path.join(python_prefix, "lib")
     env.Append(LIBPATH=[python_lib_path])
 
-    if nuitka_python and "pgo-link" in experimental:
```

Comment:
I removed the existing linux nuitka python PGO code because it wasn't really working properly. To apply the PGO info you need to rebuild the source files with -fprofile-use, which is already done during the python build itself. So the cpython-level PGO is already applied in libpython.a

Reply:
But what if you want the PGO for Python from running `libpython.a` inside your own binary, how is that supposed to work, if not through new `.gcda` files of your own making from running your program.

Reply:
Yes, that still works. There is a different code path for that. https://github.com/Nuitka/Nuitka/blob/b0cec73a13f4d49035abac2f535061b772d56a2d/nuitka/build/SconsCompilerSettings.py#L642
This part is one where you thought including the original python gcda files would be helpful.

Reply:
Ok, lets drop it, then, less noise too.